htcondor-ce/ProbeConfig.add

```diff
@@ -0,0 +1,11 @@
+
+    DataFolder="/var/lib/condor-ce/gratia/data"
+    WorkingFolder="/var/lib/condor-ce"
+    LogFolder="/var/log/condor-ce/gratia"
```

Comment:
We should create this dir in the packaging

Reply:
Right!  Adding...